src/Symfony/Component/Console/Command/Command.php

```diff
@@ -255,7 +257,7 @@ public function run(InputInterface $input, OutputInterface $output)
             $statusCode = $this->execute($input, $output);
 
             if (!\is_int($statusCode)) {
-                @trigger_error(sprintf('A non numeric or nullable $statusCode returned by Command::execute() is deprecated since Symfony 4.4, return an integer value instead.'), E_USER_DEPRECATED);
+                throw new UnexpectedValueException(sprintf('A non numeric or nullable $statusCode returned by Command::execute() is not allowed anymore since Symfony 5.0, return an integer value instead.'));
```

Comment:
I'd suggest throwing a `TypeError` instead, to prepare for real return types
the message should be future-oriented also.

`throw new \TypeError(sprintf('Return value of "%s::execute()" must be of the type int, %s returned.', \get_class($this), \gettype($statusCode)));`

Reply:
fixed